### PR TITLE
Formatting cleanup for infra_{a*, c*} via yamllint

### PR DESCRIPTION
**Pull request by ritesh97-rh**
##### SUMMARY
For the specified roles:
- Created a yamllint file to enforce formatting rules
- Made changes to existing code base to adhere to formatting requirements
- yamllint errors will be displayed during formatting checks in future development

##### ISSUE TYPE
- Code Cleanup Request

##### COMPONENT NAME
infra-aws-byo 
infra-aws-byo_verify
infra-aws-get-sandbox
infra-aws-release-sandbox
infra-aws-sandbox
infra-azure-create-inventory
infra-azure-create-service-principal
infra-azure-delete-service-principal
infra-azure-disk-map role
infra-azure-ssh-key 
infra-azure-template-destroy
infra-aws-dry-run role
infra-common-ssh-config-generate

### CODE REVIEW

File: ansible/roles-infra/infra-aws-sandbox/defaults/main.yml [hunk 1]
```diff
@@ -35,14 +39,14 @@ nuke_sandbox: true
...
-  - 809721187735 # GPTE infrastructure
-  - 384299329206 # GPTE ILT
+  - 017310218799  # Master account
+  - 627202319003  # OPENTLC Events
+  - 962799139175  # OPENTLC
+  - 719622469867  # GPE (RHPDS prod)
+  - 550201621713  # openshift BU
+  - 124572886817  # AWS dev/test
+  - 809721187735  # GPTE infrastructure
+  - 384299329206  # GPTE ILT
```
Comment (jkupferer) [inline ansible/roles-infra/infra-aws-sandbox/defaults/main.yml:49]: What is this block which adds space to each of these lines?

File: ansible/roles-infra/infra-common-ssh-config-generate/tasks/main.yml [hunk 2]
```diff
@@ -38,18 +38,18 @@
...
+         IdentityFile {{ssh_key|default(infra_ssh_key)|default(ansible_ssh_private_key_file)|default(default_key_name)}}
+         IdentitiesOnly true
+         User {{ remote_user }}
+         ControlMaster auto
+         ControlPath /tmp/{{ guid }}-%r-%h-%p
+         ControlPersist 5m
+         StrictHostKeyChecking false
+         ConnectTimeout 60
+         ConnectionAttempts 10
+         UserKnownHostsFile {{ansible_known_host}}
```
Comment (jkupferer) [inline ansible/roles-infra/infra-common-ssh-config-generate/tasks/main.yml:52]: The content we want in the SSH config should read like:
```
Host ...
  Hostname ...
  IdentityFile...
...
```
It does look like there is a YAML lint indent issue, I think it should read:
```
     content: |
￼      Host {{ bastion_hostname }} {{ hostvars[bastion_hostname].shortname |d('')}}
￼        Hostname {{ hostvars[bastion_hostname].public_dns_name }}
```

**Review by jkupferer — CHANGES_REQUESTED** [review]

File: ansible/roles-infra/infra-aws-sandbox/defaults/main.yml (hunk 1 above)
Comment (ritesh97-rh) [reply ansible/roles-infra/infra-aws-sandbox/defaults/main.yml:49]: yamllint requires two spaces for inline comments (text before comment.

File: ansible/roles-infra/infra-common-ssh-config-generate/tasks/main.yml (hunk 2 above)
Comment (ritesh97-rh) [reply ansible/roles-infra/infra-common-ssh-config-generate/tasks/main.yml:52]: Fixed

File: ansible/roles-infra/infra-aws-sandbox/defaults/main.yml [hunk 3]
```diff
@@ -24,9 +24,13 @@ output_dir: '/tmp/output_dir'
...
+  ssh-rsa
+  AAAAB3NzaC1yc2EAAAADAQABAAABAQCvZvn+GL0wTOsAdh1ikIQoqj2Fw/RA6F14O347rgKdpkgOQpGQk1k2gM8wcla2Y1o0bPIzwlNy1oh5o9uNjZDMeDcEXWuXbu0cRBy4pVRhh8a8zAZfssnqoXHHLyPyHWpdTmgIhr0UIGYrzHrnySAnUcDp3gJuE46UEBtrlyv94cVvZf+EZUTaZ+2KjTRLoNryCn7vKoGHQBooYg1DeHLcLSRWEADUo+bP0y64+X/XTMZOAXbf8kTXocqAgfl/usbYdfLOgwU6zWuj8vxzAKuMEXS1AJSp5aeqRKlbbw40IkTmLoQIgJdb2Zt98BH/xHDe9xxhscUCfWeS37XLp75J
 
-ocpkey: ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC8c3m39CoyA3bbgQUui3MGlJOryfg98NwIj8UUOv+PCuTRiyNanZeirRnPzWBtPzrVXeqtfenJo1ehDt0SRoqlUvpE7IUgnJsM7XJaUmKiVJMI4WslgMQMsZQGol7n2h0jwvtXIQp44jTtX61G7HMIUoFjJNXCO5GCrj3w2SVAKMYcznKUXpVmrkjdiHTZHmDxArHKZ7Kew2jCNUMgEq+mcaINPovUybjD17rCiKJDV4JQvLFQH89BFTVkVBsc0+QAe9LoW6l0rMDSemTvFeHy7tD0qGxYehvnHreOH6Nn9j9iOlG0f4R9+M+8KaI3uqibjOhn/kOQLK/1mphg2NnJ
+ocpkey: >-
+  ssh-rsa
+  AAAAB3NzaC1yc2EAAAADAQABAAABAQC8c3m39CoyA3bbgQUui3MGlJOryfg98NwIj8UUOv+PCuTRiyNanZeirRnPzWBtPzrVXeqtfenJo1ehDt0SRoqlUvpE7IUgnJsM7XJaUmKiVJMI4WslgMQMsZQGol7n2h0jwvtXIQp44jTtX61G7HMIUoFjJNXCO5GCrj3w2SVAKMYcznKUXpVmrkjdiHTZHmDxArHKZ7Kew2jCNUMgEq+mcaINPovUybjD17rCiKJDV4JQvLFQH89BFTVkVBsc0+QAe9LoW6l0rMDSemTvFeHy7tD0qGxYehvnHreOH6Nn9j9iOlG0f4R9+M+8KaI3uqibjOhn/kOQLK/1mphg2NnJ
```
Comment (fridim) [inline ansible/roles-infra/infra-aws-sandbox/defaults/main.yml:33]: That looks weird

i think we can locally disable:

```
# yamllint disable rule:line-length
```
for ssh pub keys entries

Comment (ritesh97-rh) [reply ansible/roles-infra/infra-aws-sandbox/defaults/main.yml:33]: Sounds good, will do!

**Review by fridim — APPROVED** [review]

**Pull request merged**